pkg/crypto/hash/hash.go

```diff
@@ -15,7 +15,7 @@ type Algorithm uint16
 
 // Supported hash algorithms
 const (
-	MD2     Algorithm = 0 // Blacklisted
+	None    Algorithm = 0 // Blacklisted
```

Comment:
Should we not keep MD2?

Reply:
I don't think this was correct before [IANA Hash Assignments](https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-18)

This showed up in bbee407bcfbf6d6a574842e09b50a5f4b67c071e and we have just carried it ever since.